packages/web/tests/pages/app/user/[uid].test.tsx

```diff
@@ -19,37 +19,61 @@ jest.mock('next/router', () => ({
 }));
 
 const useRouter = jest.spyOn(require('next/router'), 'useRouter');
+const realUseState = React.useState;
+const mockState: User = {
+  id: "123",
+  name: "Bill Sigh",
+  pronouns: "he/him",
+  schoolName: "School 5111019"
+};
 
-const sampleUser: UserProfileProps = {
-  name: 'Steve Job',
-  pronouns: 'he/him',
-  schoolName: 'Apple University',
+const sameSampleUser: UserProfileProps = {
+  isCurrentUser: true,
+  // setUser: jest.spyOn(React, 'useState').mockImplementationOnce(() => realUseState(mockState)),
```

Comment:
I am currently in the process of figuring out how to mock the state updating functions as my components have them passed in as props. I will have it done as soon as possible. I apologize for being late.

Reply:
No worries, thanks for letting us know 👍